app/assets/stylesheets/layout/_breadcrumb.sass

```diff
@@ -77,6 +79,7 @@ ul.breadcrumb
     text-decoration: none
     display: block
     width: 25px
+    font-size: 12px
```

Comment:
@floriank we may want to increase this size slightly, but that shouldn't stop this from being merged